### DISCUSSION TITLE
Separate sort type for multiple accounts

### DISCUSSION THREAD

**Discussion by jcperil** [Docile-Alligator/Infinity-For-Reddit#1252] [Duplicate Ideas] [answered]
If I have multiple accounts, type of sorting posts on one account is reflected on other accounts too. For example, **Top (Yearly)** selected on one account is reflected on all accounts. Is there a way to avoid this and have separate sorting type for different accounts?

**Comment by Wladefant** [comment, marked as answer]
Seems like you filed a bug report for that #1306 (which is correct)
